### PR TITLE
58 enable deletion of riddles

### LINKED ISSUE #58: enable deletion of riddles

### PR DESCRIPTION
**Pull request by larsuzh**
if a user is inside his own profile, the delete button is displayed
image
when pressed, an alert is displayed
image

### CODE REVIEW

File: src/app/shared/location-riddle-post/location-riddle-post.component.html [hunk 1]
```diff
@@ -13,9 +13,24 @@ <h2>{{ locationRiddleState.locationRiddle()!.username }}</h2>
...
+                ></ion-icon>
+              </ion-button>
+              <ion-alert
+                [trigger]="'present-alert-' + locationRiddle().locationRiddleId"
+                header="Are you sure you want to delete this riddle?"
+                [buttons]="alertButtons"
+              ></ion-alert>
+            }
 					</div>
 				</ion-item>
```
Comment (flyingbeat) [inline src/app/shared/location-riddle-post/location-riddle-post.component.html]: I would move the whole if statement outside of the div (inside the ion-item and add slot="end" to the button

File: src/app/shared/location-riddle-post/location-riddle-post.component.scss
Comment (flyingbeat) [inline src/app/shared/location-riddle-post/location-riddle-post.component.scss:1]: this commit: 802a4cebfd7d1919d24ab452a32b4504c0d26a6a is not necessary you can simply remove 
--color: white 

from the ion-item styles. That was wrongly set by me 😞

**Review by flyingbeat — DISMISSED** [review]
changes to fix light mode are not necessary (see comment), and position of trash icon can be improved otherwise looks great!

File: src/app/shared/location-riddle-post/location-riddle-post.component.html (hunk 1 above)
Comment (flyingbeat) [reply src/app/shared/location-riddle-post/location-riddle-post.component.html]: maybe we could even remove the dummy profile picture so there is enough space

**Comment by larsuzh** [comment]
image

**Pull request merged**